pkg/registry/servicecatalog/clusterserviceclass/storage.go

```diff
@@ -144,8 +146,32 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage) {
 		CreateStrategy: clusterServiceClassRESTStrategies,
 		UpdateStrategy: clusterServiceClassRESTStrategies,
 		DeleteStrategy: clusterServiceClassRESTStrategies,
-		Storage:        storageInterface,
-		DestroyFunc:    dFunc,
+
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "External name", Type: "string"},
```

Comment:
Seems like this should be `ExternalName`

Reply:
Changed to `External-Name` so it gets printed as `EXTERNAL-NAME` (otherwise, it would be printed as `EXTERNALNAME`).